extensions/wikia/ARecoveryEngine/ARecoveryEngine.setup.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * 	Recovery Engine - AdBlock Detection and AD recovery
+ */
+$wgAutoloadClasses['ResourceLoaderAdEngineSourcePointBase'] = __DIR__ . '/ResourceLoaders/ResourceLoaderAdEngineSourcePointBase.php';
+$wgAutoloadClasses['ResourceLoaderAdEngineSourcePointDetectionModule'] = __DIR__ . '/ResourceLoaders/ResourceLoaderAdEngineSourcePointDetectionModule.php';
+$wgAutoloadClasses['ResourceLoaderAdEngineSourcePointCSBootstrap'] = __DIR__ . '/ResourceLoaders/ResourceLoaderAdEngineSourcePointCSBootstrap.php';
```

Comment:
Urgh... yeah, `ResourceLoaderAdEngineSourcePointCipheredAndSteganographicBootstrap` would be a little too much then...